manifests/g/getzola/zola/0.20.0/getzola.zola.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with WinGet Updater using komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: getzola.zola
+PackageVersion: 0.20.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: zola.exe
+  PortableCommandAlias: zola
+ReleaseDate: 2025-02-14
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
@stephengillie you are probably right, but there is no dependency in prev. versions https://github.com/microsoft/winget-pkgs/blob/5ab03b4f902bcc55be0ecbf6f488b2aa8b89d2a8/manifests/g/getzola/zola/0.19.2/getzola.zola.installer.yaml
I'm sure that VC Redist is required for every version of Zola. I remember hitting this https://github.com/microsoft/winget-pkgs/pull/79060

I wonder how https://github.com/microsoft/winget-pkgs/pull/172832 got it....